package.json

```diff
@@ -16,24 +17,26 @@
     "compression": "^1.7.2",
     "cors": "^2.8.4",
     "express": "^4.16.2",
-    "nodemailer": "^4.6.0",
     "express-validator": "^5.0.2",
+    "http-status-codes": "1.3.0",
+    "mongodb": "^3.0.4",
+    "mongoose": "^5.0.11",
     "morgan": "^1.9.0",
-    "pug": "^2.0.0-rc.4",
-    "http-status-codes": "1.3.0"
+    "nodemailer": "^4.6.0",
+    "pug": "^2.0.0-rc.4"
   },
   "devDependencies": {
     "babel-cli": "^6.26.0",
     "babel-core": "^6.26.0",
     "babel-eslint": "^8.2.2",
     "babel-preset-env": "^1.6.1",
+    "es6-template-string-loader": "^2.0.0",
     "eslint": "^4.18.1",
     "eslint-config-standard": "^11.0.0",
```

Comment:
Can we import our eslint config? Or is it already? Does not belong to this PR thoug.

Reply:
It's already imported, but also already outdated again :D
I'll create a task with low prio ^^